playwright.config.ts

```diff
@@ -0,0 +1,41 @@
+import { defineConfig, devices } from '@playwright/test';
```

Comment:
nice, this is a great addition

Reply:
well -- i used playwright for this e2e tests... That's  indeed a nice solution to test the whole product an browser compatibility, but unfortunately it also has it's flaws.  Shadow DOM handling and noscript related behavior will often lead to reproducable wrong results! :(

Now, after the TS and vite translation, i'll very likely replace it by `vitest`.